src/client/containers/ContactUsPage/ContactUsPage.container.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import './ContactUsPage.styles.css';
-import Header from '../../components/Header/Header.component';
```

Comment:
deleted lines are not yet deleted for me when I check the codes locally after pulling develop